src/util.js

```diff
@@ -291,7 +291,9 @@ define([
                                     '$1' + xmlns + '"');
         }
         localForm = cleanForDiff(localForm);
-        serverForm = cleanForDiff(serverForm);
+        if (serverForm) {
```

Comment:
This was an annoyance that the overwrite warning was failing on standalone mode because serverForm was null. Not really related to this PR